tst/TurnConnectionFunctionalityTest.cpp

```diff
@@ -310,17 +381,17 @@ namespace com { namespace amazonaws { namespace kinesis { namespace video { name
         turnChannelDataCount = ARRAY_SIZE(turnChannelData);
         EXPECT_EQ(STATUS_SUCCESS, turnConnectionHandleChannelDataTcpMode(pTurnConnection, channelMsg, 120, turnChannelData, &turnChannelDataCount));
         EXPECT_EQ(turnChannelDataCount, 1);
-        EXPECT_EQ(turnChannelData[0].size, ARRAY_SIZE(channelData1));
-        EXPECT_EQ(0, MEMCMP(turnChannelData[0].data, channelData1, turnChannelData[0].size));
+        EXPECT_EQ(turnChannelData[0].size, ARRAY_SIZE(channelData1) - TURN_DATA_CHANNEL_SEND_OVERHEAD);
```

Comment:
There were some comments I made about the unaligned access to the buffers. In general, this might work OK as we deal with whole buffers which were lined up in the memory addresses which were aligned and perhaps it works OK. An easy test for alignment might be to feed in the buffers which are unaligned in the tests - this would trip the UndefinedBehaviorSanitizer on unaligned access/

Reply:
the buffer should follow the structure specified in the spec.